packages/tables/src/Concerns/CanSearchRecords.php

```diff
@@ -103,15 +103,23 @@ protected function applyColumnSearchesToTableQuery(Builder $query): Builder
         return $query;
     }
 
+    /**
+     * @return array<integer, string>
+     */
+    protected function searchWords(string $search): array
+    {
+        return explode(' ', preg_replace('/\s+/', ' ', trim($search)));
```

Comment:
IMO the trim should happen in `getTableSearch()`, leaving only the `preg_replace()` here

Reply:
Should I leave the trim here too? This method is dealing with white space, and if the string is wrapped in white space, then it is not handling it appropriately. `' hello world '` would give us `['', 'hello', 'world' ,'']`, four words, two of which are empty, which feels kind of strange.

Reply:
The trim should be in `getTableSearch()`, which is passed to `$search`, so the result is the same. There's no need to repeat it.

Reply:
The trim is now is `getTableSearch()`, preserving a null if present.